feat(changelog): functions-changed-serverless-functions-default-sandbox- 2024-09-19 (#3728)
scaleway/docs-content: changelog/september2024/2024-09-19-functions-changed-serverless-functions-default-sandbox-.mdx

---

> title: Serverless Functions default sandbox set V2

Reviewer: ```suggestion
title: Serverless Functions default sandbox set to v2 
```

status: changed
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-09-19
category: serverless
product: functions
---

> To reduce cold-starts we recently added a new sandbox v2.

Reviewer: ```suggestion
To reduce cold-starts, we recently added a new version of our sandbox: v2.
```

> Now on the console it's the default value for new Functions, we encorage to use it for better performances.

Reviewer: ```suggestion
This is now the default sandbox version for new Functions on the console, as it offers enhanced performance.
```
